drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -44,7 +44,6 @@ struct gpio_mcux_lpc_config {
 	IOCON_Type *pinmux_base;
 #endif
 	uint32_t port_no;
-	clock_ip_name_t clock_ip_name;
```

Comment:
Guess, you do not want to change LPC drivers in the RTxxx commit. Do you?

Reply:
I do - I guess the driver is just named because of historical reasons. The `gpio_mcux_lpc.c` driver really is used on the RT685, despite it being an i.MX RT device and not an LPC device.